### ISSUE TITLE
Consider options to make clr-debuginfo generation faster

### ISSUE THREAD

**Issue by gtkramer** [clearlinux/clr-distro-factory#92]
There are one or two find command that become expensive as debuginfo build up over time.  If we know the packages from the previous build, we might be able to identify which debuginfo packages have changed and generate debuginfo only for these in a staging directory, and then merge it into the final directory.

**Comment by phmccarty** [comment]
The other part to this issue is to implement a debuginfo lifecycle policy so that debuginfo does not "build up over time" quite like it does at the moment. Enacting such a policy would lessen the impact of running the find(1) commands.

**Comment by gtkramer** [comment]
Additionally, as a part of the merging, our syncs would no longer attempt to sync the entire directory, but rather, only place the new files into the existing directories.  This would avoid statting every file and folder in the very large directory.

**Comment by phmccarty** [comment]
Right, there are things we can do to optimize debuginfo *generation*, and we have work to do to speed up the debuginfo *syncs* as well. I'm not sure how this will work for `aws s3 sync`, but wherever `rsync` is used, we can certainly better tailor those commands for the debuginfo content.

**Comment by fenrus75** [comment]
you can delete it all once in a while

we now have like five years

so I'd say clean itnow and worry again in five years
